### ISSUE TITLE
Need resources on how to get people to use issue trackers instead of emailing bug reports

### ISSUE THREAD

**Issue by bernhold** [betterscientificsoftware/bssw.io#826]
This is for a request from Slaven Peles of the EsaSGD project.

Need resources on how to get people (in this case, within the project team) to use issue tracker to submit bug reports instead of using email.

**Comment by markcmiller86** [comment]
> Need resources on how to get people (in this case, within the project team) to use issue tracker to submit bug reports instead of using email.

Hmmm...IMHO...no way to *enforce* this. You can strongly encourage it through various means but we still get stragglers that submit poorly defined bug reports via email. Just have to deal with it. One strategy is to severely deprioritize such interactions so the user's engaging in that behavior feel enough pain to adjust their practice 😄

**Comment by bernhold** [comment]
Yes, I've seen projects where maintainers are pretty aggressive in pushing people to use their issue tracker, and follow the template there in terms of information requested with the issue.  

There can be a fine line on this.  If your project has a user mailing list, you probably need to provide some level of assistance when people report problems.  But of course often, the problem is a user error in some form or another rather than a bug in the software.

**Comment by bernhold** [comment]
I have not yet really examined Discussions.  I'm curious why you think it was worth the trouble to shift everything over and import your entire history.  If that's easier said than written, I'd be happy to chat at some point.

**Comment by markcmiller86** [comment]
Sure we can chat about that if you want. Short answer...multiple users requested ability to search/browse the email archive. And, that feature of the existing archive had fallen into disrepair.

Discussions are discoverable from search engines like Google and are searchable and browseable. It did require learning GraphQl necessary to write a python script and extensive testing before we actually did it. One side effect, our PR and issue numbers are now 5 digit numbers. They used to be 3 digit. Thats because GitHub uses a single *local* address space (e.g. numbering) for issues, prs, discussions.
